### PR TITLE
Add deriver and example of common docs

### PR DESCRIPTION
**Pull request by andypost**
This is how docs should be added as plain definition tells nothing about properties

### CODE REVIEW

File: tests/functional/Generator/_plugin_manager/_attribute/src/Attribute/Bar.php [hunk 1]
```diff
@@ -15,11 +15,24 @@ final class Bar extends AttributeBase {
 
   /**
    * Constructs a new Bar instance.
+   *
+   * @param string $id
+   *   The plugin ID. There are some implementation bugs that make the plugin
+   *   available only if the ID follows a specific pattern. It must be either
+   *   identical to group or prefixed with the group. E.g. if the group is "foo"
+   *   the ID must be either "foo" or "foo:bar".
+   * @param \Drupal\Core\StringTranslation\TranslatableMarkup $label
```
Comment (Chi-teck) [inline tests/functional/Generator/_plugin_manager/_attribute/src/Attribute/Bar.php]: `|null`

Comment (andypost) [reply tests/functional/Generator/_plugin_manager/_attribute/src/Attribute/Bar.php]: Label can't be NULL

**Comment by Chi-teck** [comment]
The PR only changes test fixture. Attribute template needs to be updated as well.
https://github.com/chesn0k/drupal-code-generator/blob/362033be04075ab3b8a09731d4c93ca6280eaf90/templates/_plugin-manager/attribute/src/Attribute/Example.php.twig

File: tests/functional/Generator/_plugin_manager/_attribute/src/Attribute/Bar.php (hunk 1 above)
Comment (andypost) [reply tests/functional/Generator/_plugin_manager/_attribute/src/Attribute/Bar.php]: set it optional

**Comment by andypost** [comment]
> The PR only changes test fixture. Attribute template needs to be updated as well. https://github.com/chesn0k/drupal-code-generator/blob/362033be04075ab3b8a09731d4c93ca6280eaf90/templates/_plugin-manager/attribute/src/Attribute/Example.php.twig

fixed

**Comment by Chi-teck** [comment]
Thank you.

**Pull request merged**